### ISSUE TITLE
is there a way to configure the tileserver address?

### ISSUE THREAD

**Issue by stepheneb** [openstreetmap/openstreetmap-website#949]
I’ve got a local instance of openstreetmap-website working and also just got a local tileserver working with a partial extract of the whole planet (a section of NJ)

I’d like to switch the osm website code to use my local tileserver however it appears to be hard-coded into several of the javascript libraries — example: vendor/leaflet/leaflet.osm.js and vendor/assets/iD/imagery.js
Is there a way to change all these references using the server_url value in config/application.yml?

**Comment by tomhughes** [comment]
No.

**Comment by ghachey** [comment]
Hi. I'm trying to achieve the exact same thing. Is it possible to use a local tile server (with an extract of OSM data) with a locally installed openstreetmap-website? I saw some places they suggest to edit vendor/assets/openlayers/OpenStreetMap.js and vendor/assets/leaflet/leaflet.osm.js. I tried that with my local tile server for what I think is the standard layer but I see nothing - just gray. The tile server seem to test ok when trying it directly. I would really appreciate any pointer in the right direction this is all pretty new to me.
